replication-api/src/main/java/com/connexta/ion/replication/api/data/ReplicatorConfig.java

```diff
@@ -108,16 +108,16 @@
   void setDescription(@Nullable String description);
 
   /**
-   * Returns the amount of times a {@link ReplicationItem} will attempt to be retried if it
-   * previously failed to replicate.
+   * Returns the amount of times a {@link com.connexta.ion.replication.api.ReplicationItem} will
+   * attempt to be retried if it previously failed to replicate.
    *
    * @return failure retry count
    */
   int getFailureRetryCount();
```

Comment:
I think you need to remove these 2 involving the failure retry count and the methods/fields in the impl